---
layout: post
title: 2nd Qiagen Genomic Tip HMW DNA Extraction of Porites lobata
category: [ Extraction ]
tags: [ HMW DNA, Porites, DNA Extraction ]
---

# 2nd Try HMW DNA Extraction for Pac-Bio Sequencing of _Porites lobata_ Flash Frozen Tissue Chunk

In this sample processing I use the [QIAGEN Genomic-tip 100/G](https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/genomic-dna/qiagen-genomic-tip-100g/#orderinginformation), the [QIAGEN Genomic DNA Buffer Set](https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/genomic-dna/blood-and-cell-culture-dna-midi-kit/#orderinginformation), [QIAGEN RNase A (100mg/mL concentration)](https://www.qiagen.com/us/products/discovery-and-translational-research/lab-essentials/enzymes/rnase-a/?clear=true#orderinginformation), [QIAGEN Proteinase K](https://www.qiagen.com/us/products/discovery-and-translational-research/lab-essentials/enzymes/qiagen-proteinase-k/?clear=true#orderinginformation), and [DNA lo-bind tubes](https://online-shop.eppendorf.us/US-en/Laboratory-Consumables-44512/Tubes-44515/DNA-LoBind-Tubes-PF-56252.html)

The sample usedis named "Porites 6" and was flash frozen chunks of coral tissue and skeleton sent to use from Hawai'i. The species was determined to be _Porites lobata_ by Sanger sequencing using the ITS region. The primers used for species ID were:
Por ITSZ1 - 5'-TAAAAGTCGTAACAAGGTTTCCGTA-3'
Por ITSZ2 - 5'-CCTCCGCTTATTGATATGCTTAAAT-3'
The sequences analyzed using BLASTn ([output here](https://github.com/meschedl/MESPutnam_Open_Lab_Notebook/blob/master/random/ZH6JHRAV016-Alignment.txt)) indicated the closest alignment to _Porites lobata_. Sanger sequencing on the ITS region and DNA prep was performed by Crawford Drury and Eva Majerová at the Gates Coral Lab at the Hawai'i Institute of Marine Biology.

**Goal**: Get 25ug of HMW DNA from Porites 6 flash frozen chunks sent from HI, the [the previous attempt](https://meschedl.github.io/MESPutnam_Open_Lab_Notebook/pm-hmw/) yielded a lot of DNA but it was not very good quality. My goal is to get sufficient DNA and better quality
**Result**: I only got 13.7ug of DNA, and the quality is pretty poor
**Major Take Aways**: We will likely have to use the previous extraction material for sequencing, because the quality is actually better (larger proportion of HMW DNA). I reduced the amount of input tissue because I thought that might be why the quality was not great on the first extraction, but that may not be the case. This time with less tissue I did not get enough DNA and the quality is worse. I am not sure if it was the method or the piece of coral that is responsible for the degradation. It is interesting that I noticed some pink in the coral chunk (mostly in the skeleton), I'm not sure if that has any quality implications but it was strange. You can only sort of see it in the images.


### Sample Processing

**Set up**

- Prepared digestion buffer with 9.5mL of Buffer G2 and 19ul of RNase A
- Set incubator genie to 50 degrees C
- Cleaned forceps, and scraper/scooper with 10% bleach, DI water, then 70% EtOH
- Placed tools and mortar and pestle in -80
- Filled Thermoflask dewar with LN2
- Brought over scale to bench

**Grinding and Incubation**

- Using sample Porites 6, flash frozen chunks in a 50mL conical
- Picked the smallest chunk with reasonable tissue (there was a smaller chunk but it had little tissue). I decided to grind this one but only use ~2/3 of the ground powder to reduce the amount of tissue used (hopefully to get better quality DNA)
- Tared the scale with the chilled mortar on it
- Put in the chunk and weighed: 1.15g
![1](https://raw.githubusercontent.com/meschedl/MESPutnam_Open_Lab_Notebook/master/images/IMG_4427.jpg)
- Poured LN2 into mortar and let boil off
- Ground chunk until it was a powder. It took a while because of the skeleton present
![2](https://raw.githubusercontent.com/meschedl/MESPutnam_Open_Lab_Notebook/master/images/IMG_4428.jpg)
- Scrapped about 2/3 of it into a chilled 50mL conical with the scraper. This was the amount left in the mortar, which I scrapped back into the sample conical
![3](https://raw.githubusercontent.com/meschedl/MESPutnam_Open_Lab_Notebook/master/images/IMG_4429.jpg)
- Poured in the buffer G2 mix
- Vortexed briefly
- Added 500ul of Proteinase K and vortexed again
- Put the conical in the incubator genie at 50 degrees C rocking 15 speed for 2 hours

### Genomic Tip Extraction

**Genomic Tip**

- Set centrifuge to 4 degrees C
- After incubation, transferred 1mL of lysed tissue liquid into each of 9 1.5mL tubes with wide bore pipette tips
- Centrifuged at 4 degrees C for 10 minutes at 5000 rcf to pellet any extra stuff
- Set up tip (resin column) inside a holder over a 50mL conical
- While that was going, added 4mL of equilibration buffer (QBT) to the tip and let drip through to the conical (took the 10 min)
- Added the supernatant from the sample tubes to the resin tip with  wide bore pipette tip and let drip through
- Dripping took ~25 minutes
- Changed 50mL waste conicals
- Added 7.5mL of buffer QC (wash) and let drip through (30min)
- Pipetted 5mL of buffer QF into a 5mL tube and warmed in incubator genie to 50 degrees C
- Repeated wash addition (30 minutes)
- Transferred resin tip to a different 50mL conical
- Added the 5mL of warmed buffer QF and let drip through (20 minutes)

**Isopropanol Precipitation of DNA**

- Made 7 DNA lo-bind 1.5mL tubes, 6 each with 833ul of the elution liquid
- Added 583ul (0.7 volumes) of room temp 100% isopropanol to each of the tubes
- Gently inverted to mix
- Centrifuged all lo-bind tubes at 10,000 rcf for 30 minutes at 1 degree C
- I forgot to put the 70% EtOH in the freezer, so after the 30 minutes I centrifuged them for another 10 minutes at 10,000 rcf at 4 degrees C while the ethanol was in the -20
- Once finished, looked for pellets: there was a visible small brown pellet in each tube
- One tube at a time, removed most of the supernatant
- One tube at a time, added 1mL of cold 70% EtOH and vortexed briefly
- Centrifuged tubes for 10 minutes at 4 degrees C 10,000 rcf
- Removed all of supernatant when finished, used a p20 to get small volumes out
- Let tubes air dry ~7 minutes
- Added 50ul of TE buffer to each of the 7 tubes very gently
- Incubated for 1hr at 55 degrees C in the Theremomixer no shaking
- Once done, transferred to an orbital shaker 100rpm at room temp for ~1 hour
- Then transferred to the 4 degree fridge for the weekend

### QC 20210208

**Qubit**

- Broad Range Qubit Monday ([protocol](https://github.com/meschedl/PPP-Lab-Resources/blob/master/Protocols/Qubit-Assay-Protocol.md))
- Flicked tube and took from both top T and bottom B of the tube measurements

|Standard 1|Standard 2|Sample|Average DNA ng/µl| Averaged top and bottom ng/ul|
|---|----|----|----|---|
|179 RFU|20822 RFU|8 T|41.8||
|-|-|8 B|60.1|50.95|
|-|-|9 T|43.4||
|-|-|9 B|44.4|43.9|
|-|-|10 T|36.9||
|-|-|10 B|63.5|50.2|
|-|-|11 T|37.3||
|-|-|11 B|52.9|45.1|
|-|-|12 T|43.8||
|-|-|12 B|54|48.9|
|-|-|13 T|44.1||
|-|-|13 B|60.1|52.1|


**Genomic DNA TapeStation**

- Followed Tapestation [protocol](https://meschedl.github.io/MESPutnam_Open_Lab_Notebook/DNA-Tapestation/), note the last lane is an extra sample for the Puritz lab, the lane would have been wasted (last in the tape) if not used
- [Results link](https://github.com/meschedl/MESPutnam_Open_Lab_Notebook/blob/master/tapestation_pdfs/2021-02-08%20-%2010.26.50.pdf)
- The quality is really low in these extractions, worse than the previous. While the main peak at the larger end has a larger mean than the previous extraction (more 19-20,000bp size, as compared to ~14,000 in the previous extraction), there looks to be more smearing. Or because there is so much less DNA the smearing is more pronounced.

**Total Amount of DNA from this extraction based on average top and bottom Qubit values and ~47ul of sample volume in each tube**
- 8: 2394.7ng DNA
- 9: 2063.3ng DNA
- 10: 2359.4ng DNA
- 11: 2119.7ng DNA
- 12: 2298.3ng DNA
- 13: 2448.7ng DNA
- **TOTAL: 13684.1ng DNA or 13.7ug of DNA**
